lib/rules/no-useless-rename.js

```diff
@@ -85,7 +90,9 @@ module.exports = {
             const properties = node.properties;
 
             for (let i = 0; i < properties.length; i++) {
```

Comment:
Do we use the index anymore, outside of the next line? If not, should this be turned into a for-of loop?

Reply:
Updated! Did some further refactoring while I was at it. The code looks a bit simpler now!